components/safety_board/HW/HW_can.c

```diff
@@ -0,0 +1,467 @@
+/**
+ * @file HW_can.c
+ * @brief  Source code for the CAN hardware
+ * @author Joshua Lafleur (josh.lafleur@outlook.com)
+ * @version 0.1
+ * @date 2023-01-15
```

Comment:
don't put versions or dates anywhere. These files live in a version controlled repo, these comments don't add value but do add confusion (when people forget to update them)